### ISSUE TITLE
Create email signup screen for iOS

### ISSUE THREAD

**Issue by Odrakir** [toggl/mobileapp#7004]
This needs:
- Textfields for email and password. 
- A button to sign up
- Another button to go to the email login screen
- A back button (to the onboarding)
- A way to show errors

Bind it with the email signup ViewModel

#7003 should be done first.

Designs and full specs: https://www.notion.so/toggl/Designs-wip-d767ebc157c542c790e1bc93eeaf006f#835299ea40344044b2572f356b4f47c1

**NOTE:** This issue is part of the `Onboarding refactor` project and it should be merged into the project branch and not into develop.

**Comment by daividssilverio** [comment]
Closed via #7134
